docs/tables/azure_sql_managed_instances.md

```diff
@@ -0,0 +1,43 @@
+
+# Table: azure_sql_managed_instances
+ManagedInstance an Azure SQL managed instance
+## Columns
+| Name        | Type           | Description  |
+| ------------- | ------------- | -----  |
+|subscription_id|text|Azure subscription id|
+|identity_principal_id|uuid|The Azure Active Directory principal id|
+|identity_type|text|The identity type|
+|identity_tenant_id|uuid|The Azure Active Directory tenant id|
+|sku_name|text|The name of the SKU, typically, a letter + Number code, eg|
+|sku_tier|text|The tier or edition of the particular SKU, eg|
+|sku_size|text|Size of the particular SKU|
+|sku_family|text|If the service has different generations of hardware, for the same SKU, then that can be captured here|
+|sku_capacity|integer|Capacity of the particular SKU|
+|provisioning_state|text|Possible values include: 'ProvisioningState1Creating', 'ProvisioningState1Deleting', 'ProvisioningState1Updating', 'ProvisioningState1Unknown', 'ProvisioningState1Succeeded', 'ProvisioningState1Failed'|
+|managed_instance_create_mode|text|Specifies the mode of database creation|
+|fully_qualified_domain_name|text|The fully qualified domain name of the managed instance|
+|administrator_login|text|Administrator username for the managed instance|
+|administrator_login_password|text|The administrator login password (required for managed instance creation)|
```

Comment:
would maybe not fetch this field `administrator_login_password` , although I am pretty sure it doesn't get fetched in read.